x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/reducer.ts

```diff
@@ -343,7 +349,7 @@ export const endpointListReducer: StateReducer = (state = initialEndpointPageSta
             activityLog,
             hostDetails: {
               ...state.endpointDetails.hostDetails,
-              detailsLoading: true,
+              detailsLoading: isNotLoadingDetails ? false : true,
```

Comment:
can this just be `!isNotloadingDetails`?

Reply:
Yes 😆 simpler.

Reply:
done 8253ccdd9e4bcb06de841758ad2166f4a2c0b779